### PR TITLE
Remove lock files on reboot not needed

### PR DESCRIPTION
**Pull request by tamilmani1989**
**Reason for Change**:

No need to remove lock files on reboot as like state files. This call causing unnecessary open to lock file.

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

File: cni/plugin.go
Comment (ashvindeodhar) [inline cni/plugin.go:163]: can you remove the func definition for this as we don't need this anymore? You can remove it from linux as well. ( the one in linux is just printing the reboot time )

func removeLockFileAfterReboot(plugin *Plugin) {
	rebootTime, _ := platform.GetLastRebootTime()
	log.Printf("[cni] reboot time %v", rebootTime)
}

**Review by ashvindeodhar — CHANGES_REQUESTED** [review]

File: store/json.go
Comment (jsturtevant) [inline store/json.go:188]: was this value `0o664` valid?

**Comment by jsturtevant** [comment]
xref: https://github.com/Azure/azure-container-networking/issues/774 this should resolve that issue partially

File: store/json.go
Comment (tamilmani1989) [reply store/json.go:188]: i changed to 0664 which is valid.

File: cni/plugin.go
Comment (tamilmani1989) [reply cni/plugin.go:163]: > xref: #774 this should resolve that issue partially

not sure if this fixes that issue.

**Comment by tamilmani1989** [comment]
> xref: #774 this should resolve that issue partially

not sure if this fixes that issue

File: cni/plugin.go
Comment (tamilmani1989) [reply cni/plugin.go:163]: lets have it for a while and will remove this once we move to filelock api

File: store/json.go
Comment (rbtr) [reply store/json.go:188]: @jsturtevant `0o644` is a legal (and unambiguous) representation of 0644, where the little `o` indicates octal

Comment (rbtr) [reply store/json.go:188]: @tamilmani1989 you uncorrected it, 0o644 is less ambiguous and therefore preferred

Comment (tamilmani1989) [reply store/json.go:188]: @rbtr oops. thanks for letting me know. i will revert this back

**Comment by ashvindeodhar** [comment]
lgtm, you can merge once you get approval from Evan for filemode changes

**Review by ashvindeodhar — APPROVED** [review]

File: store/json.go
Comment (jsturtevant) [reply store/json.go:188]: @rbtr thanks for the explanation!

**Review by rbtr — APPROVED** [review]

**Pull request merged**
